templates/Deepcell_Postprocessing.ipynb

```diff
@@ -14,12 +14,16 @@
    "outputs": [],
```

Comment:
<p>If images are being saved to <code>viz_dir</code>  then I don't know why this magic rm command is needed for removing tiffs from <code>deepcell_output</code>. <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/193/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>